css-viewport/Overview.bs

```diff
@@ -526,6 +526,39 @@ The <dfn export>unscaled</dfn> value of a CSS length relative to an element is t
 	The {{Window/devicePixelRatio}} of a frame is multiplied by the [=effective zoom=] inherited by its parent frame.
 </h3>
 
+<h2 id=extensions-to-the-window-interface>Extensions to the {{Window}} Interface</h2>
+
+<pre class=idl>
+partial interface Window {
+[SameObject, Replaceable] readonly attribute Viewport? viewport;
+};
+</pre>
+
+<h2 id=viewport>Viewport</h2>
+
+<h3 id="the-viewport-interface">The {{Viewport}} Interface</h3>
+
+<pre class=idl>
+[Exposed=Window]
+interface Viewport : EventTarget {
+  readonly attribute FrozenArray&lt;DOMRect> segments;
+};
+</pre>
+
+<h2 id='segments'>
+	The <dfn attribute for=Viewport>segments</dfn> property
+</h2>
+
+The segments property is an array of DOMRects that represent the dimensions of each existing viewport segment. Additonal details about the definition of a viewport segment can be found here : [[css-env-1#viewport-segments]].
+
+The {{segments}} property represent an immutable snapshot of the device {{segments}} when the value was queried.
+If the device state changes (rotation, window resize, <a spec="DEVICE-POSTURE">posture</a> change), the value previously retrieved will be invalid. Developers can listen to {{ScreenOrientation/onchange|Screen Orientation}}, {{Window/resize|Window Resize}}, or {{DevicePosture/onchange|Posture}} changes events to detect device state changes.
+
+The {{segments}} attribute must run these steps:
+1. If the {{Viewport}}'s associated {{Document}} is not <a>fully active</a>, return 0.
+2. Returns null if there is only a single viewport segment and abort these steps.
+3. Otherwise, return the {{Viewport}}'s segments array.
```

Comment:
Is there somewhere you can link to that defines the bounds of these segments? If not, spell it out here?

Reply:
I link `[[css-env-1#viewport-segments]]` which points [here](https://drafts.csswg.org/css-env-1/#viewport-segments) earlier in the paragraph. It explains the concept of segments but doesn't explicitly say what would represent the DOMRect. Should I spell out something like "each DOMRect represent the geometry (x, y, width, height) of each segment".

Reply:
Yes, and define in what coordinate space those 4 lie and how they are computed. Also add a non-normative Example or two.